custom_components/tahoma/coordinator.py

```diff
@@ -146,7 +147,7 @@ async def _async_update_data(self) -> Dict[str, Device]:
                 del self.executions[event.exec_id]
 
         if not self.executions:
-            self.update_interval = self.original_update_interval
+            self.update_interval = UPDATE_INTERVAL
```

Comment:
Does this take RTS into account?

Reply:
For RTS, we don’t update the interval, so that’s ok.